boilerplates/react-union-boilerplate-liferay-basic/liferay-theme/build.gradle

```diff
@@ -0,0 +1,30 @@
+buildscript {
+    repositories {
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
+    }
+    dependencies {
+        classpath "com.moowork.gradle:gradle-node-plugin:1.2.0"
+    }
+}
+
+apply plugin: 'com.moowork.gulp'
+
+node {
+    version = '8.9.1'
```

Comment:
Different version of node for theme and react union. Is it possible to have just one version for both? If not its ok.